students/rgpag/lesson04/trigrams.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+import random
+
+# txt file to be used
+text_in = 'sherlock_full.txt'
+f = open(text_in)
```

Comment:
It's recommended to use a `with` for opening files. This will automatically close the file when you exit the block.